lib/material-ui/theme.ts

```diff
@@ -1,7 +1,40 @@
-import { createMuiTheme } from '@material-ui/core/styles';
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core/styles';
 
-export const baseTheme = createMuiTheme({
+export let baseTheme = createMuiTheme({
```

Comment:
Suggestion so that it's clear that we're not really substituting the baseTheme later:

line 3: const baseTheme = createMuiTheme({...})
line 40: export default responsiveFontSizes(baseTheme);

Reply:
I did these changes in a commit now